src/test/java/com/github/dockerjava/core/RemoteApiVersionTest.java

```diff
@@ -0,0 +1,17 @@
+package com.github.dockerjava.core;
+
+import org.apache.commons.lang.SerializationUtils;
+import org.testng.annotations.Test;
+
+/**
+ * @author Kanstantsin Shautsou
+ */
+public class RemoteApiVersionTest {
+
+    @Test
+    public void testSerial() {
+        SerializationUtils.serialize(RemoteApiVersion.unknown());
```

Comment:
you should serialize some value and then deserialize it to check is it has equal field values